src/stactools/landsat/utils.py

```diff
@@ -98,3 +98,34 @@ def handle_antimeridian(item: Item, antimeridian_strategy: Strategy) -> None:
         item.geometry = Polygon(merged_coords)
 
     antimeridian.fix_item(item, antimeridian_strategy)
+
+
+def round_coordinates(item: Item, precision: int = 6) -> None:
+    """Rounds an Item's geometry and bbox geographic coordinates.
+
+    Any tuples encountered will be converted to lists.
+
+    Args:
+        item (Item): pystac Item.
+        precision (int): Number of decimal places for rounding. Defaults to 6,
+            which is approximately 1 cm at the equator.
+    """
+
+    def recursive_round(coordinates: List[Any], precision: int) -> List[Any]:
+        for idx, _item in enumerate(coordinates):
```

Comment:
Could we use `value` or something other than `_item` for this variable?

Reply:
Yeah, `_item` is craptastically bad. Changed to `value`.